packages/lesswrong/components/posts/PostsPage/PostsPage.tsx

```diff
@@ -634,7 +634,7 @@ const PostsPage = ({post, eagerPostComments, refetch, classes}: {
   </AnalyticsContext>);
 }
 
-function isDialogueParticipant(user: UsersCurrent, post: PostsDetails) {
```

Comment:
I wanna flag that this is a duplicate of the function `isUserDialogueParticipant` in the typingIndicatorsResolvrs.ts file. I'll followup today on removing the extra version of the function but meanwhile seems good to merge this.